source/backend/api/Models/Concepts/Property/PropertyModel.cs

```diff
@@ -222,6 +222,11 @@ public class PropertyModel : BaseModel
         /// </summary>
         public string GeneralLocation { get; set; }
 
+        /// <summary>
+        /// get/set - Property contacts.
+        /// </summary>
+        public List<PropertyContactModel> PropertyContacts { get; set; }
```

Comment:
nit: the other "list" fields are declared via the interface **IList** - I'd be inclined to use the interface here (instead of the concrete implementation **List**) to keep it consistent

Reply:
sure